ui/src/app/components/webcomponents/action-test-data-runtime-variable-suggestion.component.ts

```diff
@@ -219,4 +219,8 @@ export class ActionTestDataRuntimeVariableSuggestionComponent implements OnInit
     }
   }
 
+  get isStepRecorderView(){
```

Comment:
plz add comment a jsdoc comment to describe the purpose @manikanta-reddy-t

Reply:
added